src/main/java/at/hannibal2/skyhanni/config/features/garden/visitor/InventoryConfig.kt

```diff
@@ -1,37 +1,43 @@
-package at.hannibal2.skyhanni.config.features.garden.visitor;
+package at.hannibal2.skyhanni.config.features.garden.visitor
 
-import at.hannibal2.skyhanni.config.FeatureToggle;
-import com.google.gson.annotations.Expose;
-import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
-import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import at.hannibal2.skyhanni.config.FeatureToggle
+import com.google.gson.annotations.Expose
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption
 
-public class VisitorInventoryConfig {
+class InventoryConfig {
```

Comment:
please rename it back to visitlr inventory config

Reply:
![image](https://github.com/user-attachments/assets/e10c6363-dee4-4011-a93d-e2c9549e1b9a)